src/ViewModel/Converter/AnnotationAnnotationTeaserConverter.php

```diff
@@ -81,4 +81,16 @@ protected function getViewModelConverter() : ViewModelConverter
     {
         return $this->viewModelConverter;
     }
+
+    private function generateUri(Annotation $annotation) : string
+    {
+        if ($annotation->getParents()->notEmpty()) {
+            $id = $annotation->getParents()[0];
```

Comment:
I guess all the annotations in the chain are listed. Perhaps this should have been called `ancestors`

Reply:
Yep.

Reply:
They seem to be called `references` in the Hypothesis API, so we should definitely rename.

Reply:
ancestors++